1725388227783	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1725418183946	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1725418183974	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1725418183998	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1725418184026	Sync.SyncScheduler	DEBUG	Woke from sleep.
1725418184028	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1725418184086	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1725418184086	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1725418184088	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1725418186090	Sync.Service	DEBUG	User-Agent: Firefox/130.0 (Linux x86_64) FxSync/1.132.0.20240902145003.desktop
1725418186090	Sync.Service	INFO	Starting sync at 2024-09-04 08:19:46 in browser session pPvMiEO43pN9
1725418186090	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1725418186097	Sync.Status	INFO	Resetting Status.
1725418186097	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1725418186098	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1725418186098	FirefoxAccounts	DEBUG	getOAuthToken enter
1725418186099	FirefoxAccounts	TRACE	getCachedToken returning cached token
1725418186099	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1725418186099	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1725418186100	Sync.SyncAuthManager	INFO	Getting sync key
1725418186101	FirefoxAccounts	DEBUG	getOAuthToken enter
1725418186101	FirefoxAccounts	TRACE	getCachedToken returning cached token
1725418186101	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1725418186101	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1725418186101	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1725418186101	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1725418186101	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1725418186106	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1725418186106	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1725418186107	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1725418186107	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1725418186108	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1725418186108	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1725418186108	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1725418186108	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1725418186110	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1725418186110	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1725418186113	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/261848012/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725418186113	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1725418186113	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1725418186114	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725418186114	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1725418186115	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1725418186125	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
